src/components/common/Reactions.tsx

```diff
@@ -347,10 +358,41 @@ export const Reactions: React.FC<{
               </UniLink>
             </Trans>
           </div>
-          <div className="h-16 border-t flex items-center px-5">
+          <div className="border-t flex flex-col md:flex-row gap-4 items-center px-5 py-4">
             <Button isBlock onClick={() => setIsLikeOpen(false)}>
               {t("Got it, thanks!")}
             </Button>
+            <Button
+              variant="secondary"
+              isBlock
+              onClick={() => {
+                setIsUnlikeOpen(true)
+                setIsLikeOpen(false)
+              }}
+            >
+              {t("Revert")}
+            </Button>
+          </div>
+        </Modal>
+        <Modal
+          open={isUnlikeOpen}
+          setOpen={setIsUnlikeOpen}
+          title={t("Confirm to revert")}
+        >
+          <div className="p-5">
+            <Trans i18nKey="like revert">
+              Do you really want to revert this like action? <br />
+              Please note that this is also an on-chain operation, your reversal
+              will have a transaction too.
```

Comment:
It's not good to put pressure on users, I think it's better to remove `Please note that...`.